graphite_api/app.py

```diff
@@ -578,6 +589,8 @@ def tree_json(nodes, base_path, wildcards=False):
 
 
 def doImageRender(graphClass, graphOptions):
+    if not cairo:
```

Comment:
Was thinking it would be better not to fail actually. Since we do want it to fail loudly, will remove that check and let the underlying exception get raised.